Controllers/GameController.cs

```diff
@@ -1,94 +1,113 @@
 using Harmonify.Data;
+using Harmonify.Models;
 using Harmonify.Responses;
 using Harmonify.Services;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Primitives;
 
-namespace Harmonify.Controllers
+namespace Harmonify.Controllers;
+
+[ApiController]
+public class GameController(IGameService gameService, IWebSocketService webSocketService)
+  : ControllerBase
 {
-  [ApiController]
-  public class GameController(
-    IGameRepository gameRepository,
-    IPlayerRepository playerRepository,
-    IWebSocketService webSocketService
-  ) : ControllerBase
+  [ApiExplorerSettings(IgnoreApi = true)]
+  [Route("create")]
+  public async Task CreateGame()
   {
-    readonly IGameRepository gameRepository = gameRepository;
-    readonly IPlayerRepository playerRepository = playerRepository;
-
-    [HttpPost("create")]
-    public ActionResult CreateGame()
+    if (!HttpContext.WebSockets.IsWebSocketRequest)
```

Comment:
Może dałoby się tego if'a z returnem wyodrębnić do funkcji bo ten blok pojawia się ze 3 razy

Reply:
Dałem approve'a, bo nie jestem pewien czy tego if'a powinno się wyodrębnić, więc aby nie blokować pulla, jeżeli nie jest to do zmiany.

Reply:
Ja już wyodrębniłem to co jest w środku if'a, dodanie jego zaoszczędziłoby jedną linijkę a nazwanie takiej funkcji byłoby w chuj niezręczne